lib/commit_after_review.js

```diff
@@ -0,0 +1,76 @@
+function commitAfterReview(commit, review, logger) {
+  let commitIndex = commit.length === 0
+    ? 0 : commit.length - 1;
+  let reviewIndex = review.length === 0
+    ? 0 : review.length - 1;
+
+  let lastCommit = commit[commitIndex].commit;
+  let lastReview = review[reviewIndex];
+
+  let commitInfo = getDate(lastCommit.committedDate);
+  let reviewInfo = getDate(lastReview.publishedAt);
```

Comment:
If `review.length === 0` then this will throw

Reply:
The easiest way to grab all oudated reviews would be 

```
const lastCommit = commits.sort((a, b) => descening(a.committedDate, b.committedDate))[0];
if (lastCommit) {
  const oudatedReviews = reviews.filter((r) => r.publishedAt < lastCommit.committedDate);
  // iterate through the outdatedReviews and warn for each
}
```

Reply:
The one concern I have in general with making this 'super strict' is that currently most people aren't very good about affirming their reviews and often squashes happen with no one coming back to confirm that they still approve. If we log every single review that's outdated then that seems like it might be a tad too much.

I feel like maybe it's almost more helpful to warn for every commit that happened after reviews and show the commits message. That way they can assess whether the PR needs more reviews or it was just maintenance commits.

Reply:
@apapirovski Yes, I think it's more useful to warn about the commits (with their messages) instead of the reviews. If the commits are all like "fix nits" then the people landing them don't have to worry about it.